pkg/analyze/distribution.go

```diff
@@ -202,6 +214,9 @@ func analyzeDistribution(analyzer *troubleshootv1beta2.Distribution, getCollecte
 		}
 	}
 
+	result.IsWarn = true
+	result.Message = "None of the conditionals were met"
```

Comment:
How about:  "Unable to determine the Kubernetes distribution"

Reply:
I considered using such a message, but if I had something like 
fail: "== docker-desktop"
fail: "== aks"
pass: "== minikube"
and my cluster is in ibm-cloud, that message would also be misleading. Preflight was able to determine the distribution but the conditionals are not met. 

What if I add the message "Unknown distribution: xxx."? We can append that to the analyzer's message, either if it passed, failed or none of the conditions was met